packages/components/Organisms/SiteHeader/index.vue

```diff
@@ -496,6 +496,7 @@ export default {
       }
 
       img {
+        image-rendering: -webkit-optimize-contrast;
```

Comment:
I cant find anything about this style on mdn / google, what is this sorcery? Also do we need to worry about non webkit browsers?

Reply:
IKR! I found a few mentions of it on this 6 year old bug report on chromium https://bugs.chromium.org/p/chromium/issues/detail?id=562162

As far as I can tell, it only affects webkit browsers - IE and FF both seem fine (and non-blurry) already.